code/modules/client/client_procs.dm

```diff
@@ -417,6 +417,9 @@
 	if(karmaholder)
 		karmaholder.processRefunds(mob)
 
+	// Tell client about their connection
+	to_chat(src, "<span class='notice'>You are currently connected [prefs.server_region ? "via the <b>[prefs.server_region]</b> relay" : "directly"] to Paradise.</span>")
```

Comment:
So this PR doesn't allow people joining Paradise for the first time to join a relay server. Which means they'll get this misleading message saying they're connected via a relay if they have that pref set. I'm not even sure we can control which relay server they join when they first join from the hub right?

Reply:
Nope, however.
`Which means they'll get this misleading message saying they're connected via a relay if they have that pref set. `

They wont have that pref set because they're new.

Reply:
Ok my original message is not clear. What I mean is that the message will be shown *anytime* someone connects to para, which could be misleading. It makes sense to be shown for reboots, but not manual connections.

Reply:
It wont be. The proxy engine routes them properly no matter where you join from.

![image](https://user-images.githubusercontent.com/25063394/185802366-f85e8818-cd4e-4b73-b652-6c0e349a3a35.png)

You could connect to `aus.paradisestation.org` and it would funnel you to UK if you have it set.

Reply:
Oh sick. Nevermind then.